File: 8.cpp
#include <iostream>
using namespace std;

int main() 
{
    double num1, num2;
    cout << "Enter the first number: ";
    cin >> num1;
    cout << "Enter the second number: ";
    cin >> num2;
    if (num1 > num2) 
    {
        cout << "The larger number is: " << num1 << endl;
    }
    else
    {
        cout << "The larger number is: " << num2 << endl;
    }
    return 0;
}

File: 8.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});
let numbers = [];
rl.question("Enter the first number: ", function (input1) {
    numbers.push(parseFloat(input1));
    rl.question("Enter the second number: ", function (input2) {
        numbers.push(parseFloat(input2));
        const larger = numbers[0] > numbers[1] ? numbers[0] : numbers[1];
        console.log("The larger number is: " + larger);
        rl.close();
    });
});
